models/news.models.js

```diff
@@ -36,3 +27,18 @@ exports.findArticles = (sort_by = "created_at") => {
     return articles;
   });
 };
+
+exports.findCommentsByArticleId = (article_id) => {
+  return db
+    .query(
+      `SELECT comments.comment_id, comments.votes, comments.created_at, comments.author, comments.body, comments.article_id FROM comments WHERE article_id = $1 ORDER BY comments.created_at DESC;`,
+      [article_id]
+    )
+    .then(({ rows }) => {
+      const articles = rows;
+      if (articles.length === 0) {
+        return Promise.reject({ status: 404, msg: "Article not found" });
```

Comment:
What if the article is found but there are no comments? Should it still error? Should it say Article not found?